src/main/java/org/terracotta/tinypounder/KitAwareClassLoaderDelegator.java

```diff
@@ -58,14 +65,17 @@ public KitAwareClassLoaderDelegator() {
     } catch (IOException e) {
       // that's fine, it's probably just a kit without tc store
     }
-    Stream<Path> clientLibJarStream;
+    Stream<Path> clientLibJarStream = Stream.empty();
     try {
       clientLibJarStream = Files.walk(Paths.get(kitPath, "/client/lib"), 4, FileVisitOption.FOLLOW_LINKS)
           .filter(path -> path.toFile().getAbsolutePath().endsWith("jar"));
     } catch (IOException e) {
       // it's possible that the client libs are elsewhere...
-      clientLibJarStream = Files.walk(Paths.get(kitPath, "../common/lib"), 4, FileVisitOption.FOLLOW_LINKS)
-          .filter(path -> path.toFile().getAbsolutePath().endsWith("-client.jar"));
+      try {
+        clientLibJarStream = Files.walk(Paths.get(kitPath, "../common/lib"), 4, FileVisitOption.FOLLOW_LINKS)
+            .filter(path -> path.toFile().getAbsolutePath().endsWith("-client.jar"));
+      } catch (IOException ignored) {
```

Comment:
I'm not sure I agree with this...
if there's an exception here, it means no client libs were found either under client/lib nor ../common/lib ; that means it's not a kit, there's something missing.